src/txn/txn.c

```diff
@@ -881,23 +881,33 @@ __txn_commit_timestamps_usage_check(WT_SESSION_IMPL *session, WT_TXN_OP *op, WT_
           "no timestamp provided for an update to a "
           "table configured to always use timestamps once they are first used"));
 
-    if (FLD_ISSET(ts_flags, WT_DHANDLE_TS_ORDERED) && txn_has_ts && prev_op_durable_ts > op_ts)
+    /*
+     * The program should exit abnormally in addition to just logging a message.
```

Comment:
Instead of this comment, I would suggest adding something about what we are checking here.

Reply:
I'll change this to a more informative description.